libs/api/app/data-access/src/lib/api-app-webhook-data-access.service.ts

```diff
@@ -78,11 +79,18 @@ export class ApiAppWebhookDataAccessService {
         return res.send(new Error(`webhookAcceptIncoming is disabled`))
       }
 
+      if (!headers['kinetic-tx-id']) {
+        return res.send(new Error(`Error finding tx-id`))
+      }
+      const appTransactionId = headers['kinetic-tx-id'].toString()
+
+      console.log('appTransactionId, ', appTransactionId)
```

Comment:
Do we need this log?

Reply:
Nope, not needed. I just removed it in #280